modules/odc-monitoring-your-project-metrics.adoc

```diff
@@ -10,7 +10,7 @@ After you create applications in your project and deploy them, you can use the *
 
 .Procedure
 
-. On the left navigation panel of the *Developer* perspective, click *Observe* to see the *Dashboard*, *Metrics*, *Alerts*, and *Events* for your project.
+. Go to *Observe* to see the *Dashboard*, *Metrics*, *Alerts*, and *Events* for your project.
```

Comment:
What about "click" here instead of "go to". I think it is okay either way though